compiler/noirc_frontend/src/hir/type_check/errors.rs

```diff
@@ -136,6 +136,8 @@ pub enum TypeCheckError {
     UnconstrainedReferenceToConstrained { span: Span },
     #[error("Slices cannot be returned from an unconstrained runtime to a constrained runtime")]
     UnconstrainedSliceReturnToConstrained { span: Span },
+    #[error("Call to unconstrained function is unsafe and requires unconstrained function or unsafe block")]
```

Comment:
```suggestion
    #[error("Call to unconstrained function is unsafe and must be in an unconstrained function or unsafe block")]
```
What about the `unsafe fn` in the original issue? Are those implemented here?

Reply:
Do you mean being able to call unconstrained functions inside unconstrained functions without having to wrap them in `unsafe`? If it's that, it's working... that said, I don't know if there's a test for that.